src/app/server/CommissioningWindowManager.cpp

```diff
@@ -526,6 +540,20 @@ CHIP_ERROR CommissioningWindowManager::StopAdvertisement(bool aShuttingDown)
         (void) chip::DeviceLayer::ConnectivityMgr().SetBLEAdvertisingEnabled(false);
     }
 #endif // CONFIG_NETWORK_LAYER_BLE
+#if CHIP_DEVICE_CONFIG_ENABLE_WIFIPAF
+    if (mAdvertisingOverWiFiPAF)
+    {
+        ChipLogProgress(AppServer, "Stopping Wi-Fi PAF publish if connection has not established");
+        WiFiPAF::WiFiPAFSession sessionInfo = { .role = WiFiPAF::WiFiPafRole::kWiFiPafRole_Publisher };
+        WiFiPAF::WiFiPAFLayer & pafLayer    = WiFiPAF::WiFiPAFLayer::GetWiFiPAFLayer();
+        WiFiPAF::WiFiPAFSession * pSession  = pafLayer.GetPAFInfo(WiFiPAF::PafInfoAccess::kAccSessionId, sessionInfo);
+        if ((pSession != nullptr) && (pSession->peer_id == WiFiPAF::kUndefinedWiFiPafSessionId))
```

Comment:
See comments above.  Those implementation details should be hidden inside the API surface... 

Importantly: that will let us see whether we have correct behavior for the case when we want to stop advertising but we have a "PAF session" established....  How does that behave?  The device should _not_ remain discoverable if PASE over that PAF session fails, for example.  Right now it's pretty hard to tell what this is doing.